src/www/ui/api/Models/ClearingHistory.php

```diff
@@ -0,0 +1,182 @@
+<?php
+/*
+ SPDX-FileCopyrightText: © 2023 Samuel Dushimimana <dushsam100@gmail.com>
+ SPDX-License-Identifier: GPL-2.0-only
+*/
+/**
+ * @file
+ * @brief Info model
+ */
+namespace Fossology\UI\Api\Models;
+
+use EasyRdf\Literal\Date;
+
+/**
+ * @class Info
+ * @brief Info model to contain general error and return values
+ */
+class ClearingHistory
+{
+  /**
+   * @var Date $date
```

Comment:
Seeing the constructor call and other usage, it seems like the data type of `$date` should be `string` instead. (`date()` returns a string).